src/components/case_manager/PersonTable.tsx

```diff
@@ -0,0 +1,398 @@
+  const [showInput, setShowInput] = useState<boolean>(false);
+
+  const handleOpenInput = () => {
+    setShowInput(true);
+  };
+
+  const handleCloseInput = () => {
+    setShowInput(false);
+  };
+
+  const handleRequestSort = (
+    event: React.MouseEvent<unknown>,
+    property: string
+  ) => {
+    const isAsc = orderBy === property && order === "asc";
+    setOrder(isAsc ? "desc" : "asc");
+    setOrderBy(property as keyof Person);
+  };
+
+  const handleSelectAllClick = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.checked) {
+      const newSelected = people.map((n) => n.email);
+      setSelected(newSelected);
+      return;
+    }
+    setSelected([]);
+  };
+
+  const handleClick = (event: React.MouseEvent<unknown>, email: string) => {
+    const selectedIndex = selected.indexOf(email);
+    let newSelected: readonly string[] = [];
+
+    if (selectedIndex === -1) {
+      newSelected = newSelected.concat(selected, email);
+    } else if (selectedIndex === 0) {
+      newSelected = newSelected.concat(selected.slice(1));
+    } else if (selectedIndex === selected.length - 1) {
+      newSelected = newSelected.concat(selected.slice(0, -1));
+    } else if (selectedIndex > 0) {
+      newSelected = newSelected.concat(
+        selected.slice(0, selectedIndex),
+        selected.slice(selectedIndex + 1)
+      );
+    }
+
+    setSelected(newSelected);
+  };
+
+  const isSelected = (name: string) => selected.indexOf(name) !== -1;
+
+  const visibleRows = React.useMemo(() => {
+    return people.slice().sort(getComparator(order, orderBy));
+  }, [order, orderBy, people]);
+
+  return (
+    <Box sx={{ width: "75%" }}>
+      {/* Form can't be integrated with table (can't span multiple cells), so the form is on the outside with inputs referencing its id */}
+      <form onSubmit={onSubmit} id="table_input_form"></form>
+      <EnhancedTableToolbar
+        numSelected={selected.length}
+        totalRows={people.length}
+        type={type}
+        onOpenInput={handleOpenInput}
+      />
+      <TableContainer>
+        <Table
+          sx={{ minWidth: 750 }}
+          aria-labelledby={`Table of ${"students/paras"}`}
+        >
+          <EnhancedTableHead
+            headCells={headCells}
+            numSelected={selected.length}
+            order={order}
+            orderBy={orderBy as string}
+            onSelectAllClick={handleSelectAllClick}
+            onRequestSort={handleRequestSort}
+            rowCount={people.length}
+          />
+          <TableBody>
+            {showInput && (
+              <EnhancedTableInput
+                inputCells={headCells}
+                type={type}
+                onCloseInput={handleCloseInput}
+              />
+            )}
+            {visibleRows.map((row, index) => {
+              const isItemSelected = isSelected(row.email);
+              const labelId = `enhanced-table-checkbox-${index}`;
```

Comment:
🐧 is the prefix here used for anything?
```suggestion
              const labelId = row.email;
```